docs/modules/ic-identity-guide/pages/auth-how-to.adoc

```diff
@@ -6,9 +6,9 @@
 :company-id: DFINITY
 :sdk-short-name: DFINITY Canister SDK
 
-The Internet Identity enables you to authenticate securely and anonymously when you access applications that use it as an authentication method. A different identity is created for each application you log in to, and you will be able to use all of your registered devices or authentication methods to log in to the same account. This is all accomplished using encrypted smart contracts that run on the {IC} blockchain.
+The Internet Identity enables you to authenticate securely and anonymously when you access applications that use it as an authentication method. A different pseudonym is created for each application you use, and you will be able to use all of your registered devices or authentication methods to log in to the same account. This is all accomplished using smart contracts that run on the {IC} blockchain.
```

Comment:
I think "The Internet Identity" is already a weird start for the sentence. What is "the Internet Identity"?

Reply:
Agreed. I removed 'The'. Also removed it in the title and one more occurrence. 

I think this is better since 'Internet Identity' refers to the "product" that we build. 

Let me know if you have a better idea. The application/product must have a name, right, and I guess that is 'Internet Identity'?